src/test/java/bio/overture/ego/controller/TokensOnPermissionsChangeTest.java

```diff
@@ -175,7 +175,7 @@ public void downgradePermissionFromUser_ExistingToken_RevokeTokenSuccess() {
   @SneakyThrows
   public void denyPermissionFromUser_ExistingToken_RevokeTokenSuccess() {
     val user = entityGenerator.setupUser("UserFoo DenyPermission");
-    val policy = entityGenerator.setupSinglePolicy("song.abc");
+    val policy = entityGenerator.setupSinglePolicy("songabc");
```

Comment:
aw crap I forgot about this. SONG might use dots to delimit studies. Let me verify that we are not doing this in ICGC ARGO.